components/app.js

```diff
@@ -0,0 +1,72 @@
+class App {
+    constructor(gradeTable, pageHeader, gradeForm) {
+        this.handleGetGradesError = this.handleGetGradesError.bind(this)
+        this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this)
+        this.gradeTable = gradeTable
+				this.pageHeader = pageHeader
```

Comment:
Something strange is going on with your indentation, I'm going to group you up with Uzair to see what's happening.